src/routes/docs/tooling/command-line/commands/+page.markdoc

```diff
@@ -4,67 +4,164 @@ title: Commands
 description: Learn about Appwrites CLI and the powerful, feature complete commands to manage Appwrite's auth, databases, functions, storage, and more.
 ---
 
-Other than commands to create and deploy databases, collections, functions, teams, and buckets, the Appwrite CLI can be used as a Server SDK as well. The Appwrite CLI has a command for every Server API endpoint.
+{% info title="CLI Version" %}
+It is highly recommended that you have the latest version of the CLI. [Update your CLI](https://www.npmjs.com/package/appwrite-cli-beta) by running the following command in your terminal:
+```sh
+npm install -g appwrite-cli-beta@latest
```

Comment:
The upgrade command varies based on how the CLI was installed. We should probably add info about how to update on the installation page and then link to it from here.